exercises/change/change_test.rb

```diff
@@ -45,16 +45,22 @@ def test_no_coins_make_0_change
 
   def test_error_testing_for_change_smaller_than_the_smallest_of_coins
     skip
-    assert_equal -1, Change.generate([5, 10], 3)
+    assert_raises(ArgumentError) do
+      Change.generate([5, 10], 3)
```

Comment:
`ArgumentError` is probably OK here, but what about returning a subclass which is more descriptive?